files/common/var/lib/delphix-platform/ansible/10-delphix-platform/roles/delphix-platform/tasks/main.yml

```diff
@@ -246,6 +246,10 @@
     - { key: "AllowStreamLocalForwarding", value: "no" }
     - { key: "AllowTcpForwarding", value: "no" }
     - { key: "X11Forwarding", value: "no" }
+    #
+    # Configure SSH to exclude the 'ssh-rsa' algorithm from HostKeyAlgorithms
+    #
+    - { key: "HostKeyAlgorithms", value: "-ssh-rsa*" }
```

Comment:
This task is missing the "sshd config changed" notification. This means that the sshd service will not be restarted on upgrade after this change, and thus the change won't take effect immediately on upgrade as intended. Can you fix that while you're here? You can see how this notification works by looking for "sshd config changed" elsewhere in the file.

Reply:
Thanks Seb, made the changes and tested.